packages/dds/tree/src/feature-libraries/schema-index/codec.ts

```diff
@@ -22,7 +23,14 @@ import {
 } from "../../core/index.js";
 import { brand, type JsonCompatible } from "../../util/index.js";
 
-import type { Format as FormatV1 } from "./formatV1.js";
+import { Format as FormatV1 } from "./formatV1.js";
+
+/**
+ * Schema codec write version.
```

Comment:
If accurate, I'd go for something more verbose like this for extra clarity for newcomers into this part of the codebase. Based on other parts of the code in this file, maybe talking about versions of the persisted format (instead of versions of the codec) would be clearer?
```suggestion
 * Versions for the codec that encodes an in-memory representation of a stored schema into a persisted format (or decodes it in the opposite direction).
```